### DISCUSSION TITLE
Electric furnace experiment validation through FDS?

### DISCUSSION THREAD

**Discussion by nevertrustenvironment** [firemodels/fds#13647] [General]
Hello everyone!. 
I did an experiment on a concrete block inside an electric furnace to analyze the inside temperature of concrete.  The fire temperature was set according to the RWS fire curve (Rijkswaterstaat fire curve).  I want to do the numerical simulation through FDS to study the different variables.  Is it possible to set the fire temperature in FDS? I want to apply the surface temperature of concrete from the experiment as a fire curve for the simulation.

**Comment by mcgratta** [comment]
You can specify the wall temperature of the oven, not the concrete block, to follow the RWS curve.

**Reply by nevertrustenvironment** [reply]
> You can specify the wall temperature of the oven, not the concrete block, to follow the RWS curve.

Thank you for the suggestions. How can we specify the oven wall temperature during FDS simulation? Would you help me to provide some references? Thank you.

**Comment by Er9y714** [comment]
Note that, the convective heat transfer to the concrete element also depends on the velocity. In test furnaces the burners supply a premixed fuel-air with a high velocity. So the air in the furnace is non stagnant. Applying different or no forced air flow will have an effect on the heat transfer.

**Reply by nevertrustenvironment** [reply]
> he convective heat transfer to the concrete element also depends on the velocity. In test furnaces the burners supply a premixed air with a high velocity. So the air in the furnace is non stagnant. Applying different or no forced air flow will have an effect on the

Thank you for the suggestions.

**Reply by mcgratta** [reply]
OK, you can force hot air into the furnace too.
